task36.py

```diff
@@ -0,0 +1,8 @@
+def print_operation_table(operation, num_rows=6, num_columns=6):
+    for i in range(1, num_rows+1):
+        row = []
+        for j in range(1, num_columns+1):
+            row.append(str(operation(i, j)))

```

Comment:
У вас таблица не выровнена. Такой вариант, решит эту проблему.
```suggestion
            row.append(str(operation(i, j)).rjust(len(str(operation(num_rows, j)))))
```
Вариант ужасный, с точки зрения кода, но работу свою делает при любой функции `operation`